TWLight/applications/views.py

```diff
@@ -1161,6 +1161,16 @@ class SendReadyApplicationsView(PartnerCoordinatorOnly, DetailView):
     model = Partner
     template_name = "applications/send_partner.html"
 
+    def dispatch(self, request, *args, **kwargs):
+        partner = self.get_object()
+        auth_method = partner.authorization_method
+        if auth_method == Partner.EMAIL or auth_method == Partner.CODES:
+            return super(SendReadyApplicationsView, self).dispatch(
+                request, *args, **kwargs
+            )
+        else:
+            raise Http404("Application's for this Partner are sent automatically")
```

Comment:
Just one small typo - "Application's" should be "Applications".

Reply:
fixed it :)